code/api/src/modules/product/query.test.js

```diff
@@ -2,21 +2,109 @@ import request from 'supertest'
 import express from 'express'
```

Comment:
Had to run `NODE_ENV=test npm run setup:db` to add migrations for the crate test database